recipes/cmake/3.x.x/conanfile.py

```diff
@@ -14,6 +14,8 @@ class CMakeConan(ConanFile):
     generators = "cmake"
     settings = "os", "arch", "compiler", "build_type"
 
+    requires = "openssl/1.1.1g"
```

Comment:
openssl isn't enabled by default on Windows, I think this requires needs to be os specific

Reply:
I'll have to add the `CMAKE_USE_OPENSSL` option.